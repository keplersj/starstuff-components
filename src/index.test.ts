import * as Components from ".";

describe("Starstuff Components Library", () => {
  it("contains the correct list of exported components", () => {
    const componentList = Object.keys(Components);

    expect(componentList).toMatchSnapshot();
  });
});
