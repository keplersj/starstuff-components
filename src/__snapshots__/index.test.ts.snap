// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Starstuff Components Library contains the correct list of exported components 1`] = `
Array [
  "Card",
  "Hyperbutton",
  "Hyperlink",
]
`;
